src/main/java/net/coderbot/iris/gui/GuiUtil.java

```diff
@@ -0,0 +1,83 @@
+package net.coderbot.iris.gui;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.DrawableHelper;
+import net.minecraft.client.sound.PositionedSoundInstance;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.sound.SoundEvents;
+import net.minecraft.util.Identifier;
+
+/**
+ * Class serving as abstraction and
+ * centralization for common GUI
+ * rendering/other code calls.
+ *
+ * Helps allow for easier portability
+ * to Minecraft 1.17 by abstracting
+ * some code that will be changed.
+ */
+public final class GuiUtil {
+	/**
+	 * Resource ID for the texture used for
+	 * Iris's GUI widgets.
+	 */
+	public static final Identifier IRIS_WIDGETS_TEX = new Identifier("iris", "textures/gui/widgets.png");
```

Comment:
This should probably just be private, and you can then have a bindWidgetsTexture function in place of bindTexture.

Reply:
Noted, however see my next few responses